test/recipes/80-test_tsa_data/sectigo-signer.pem

```diff
@@ -0,0 +1,40 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
You probably want to remove this file since the branch already rebased for the inclusion of #14504.

Reply:
You are right that this file and the following one are not directly needed anymore
since the leaf cert and the first intermediate cert are now taken from the signed response. I've just extended the comments in `80-test_tsa.t` to mention that it includes them.

Yet I deliberately left those files in for the case anyone wants to have a look at them
because it's rather awkward to view/extract them from the signed response.